src/main/resources/db/migration/V0003__SCRUM-2024.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO subdomain (code, name, description)
+VALUES ('101', 'literature_reference', 'Literature Reference');
+CREATE SEQUENCE subdomain_literature_reference_seq;
+
+INSERT INTO subdomain (code, name, description)
+VALUES ('102', 'literature_resource', 'Literature Resource');
```

Comment:
Sorry didn't see the ticket but is this the name they wanted? literature_resource vs resource?

Reply:
I will use resource and reference!

Reply:
On #scrum-blueteam Andres described this value as the field used for the API only, not something that affects other things, so it seemed like it didn't matter too much.  I didn't think we needed 'literature_' in front, to match the linkml, but Ceri didn't say anything.  If everyone's good with 'resource' and 'reference' that's probably best, so I'm glad you both agree.  I don't know your internals, whether that affects the postgres column name and description, or also the sequence name.

Reply:
Ya... because in the future we are going to have `gene` and not `curation_gene` so linking the system to the name doesn't make sense to me.

Reply:
Changed!